lib/trailblazer/rails/railtie/loader.rb

```diff
@@ -1,56 +1,52 @@
+gem 'trailblazer-loader'
+
 require "active_support/concern"
+require "trailblazer/loader"
+require "trailblazer/rails"
 
 module Trailblazer
-  class Railtie < ::Rails::Railtie
-    module Loader
-      extend ActiveSupport::Concern
+  module Railtie::Loader
+    extend ActiveSupport::Concern
 
-      included do # rubocop:disable Metrics/BlockLength
-        def self.load_concepts(app)
-          load_for(app)
+    included do # rubocop:disable Metrics/BlockLength
+      gem 'trailblazer-loader'
```

Comment:
why do you need the gem here if it's added in the `Gemfile`?

Reply:
this is duplicate, i will remove it.